typing/env.ml

```diff
@@ -614,10 +626,15 @@ let sign_of_cmi ~freshen { Persistent_env.Persistent_signature.cmi; _ } =
   let loc = Location.none in
   let pm_signature = lazy (Subst.signature Subst.identity sign) in
   let pm_components =
-    let freshening_subst =
-      if freshen then (Some Subst.identity) else None in
+    let scope, freshening_subst =
+      if freshen then (
+        Ident.lowest_scope, Some Subst.identity
+      ) else
+        Ident.highest_scope, None
```

Comment:
Maybe add a comment here, as the role of `Ident.lowest_scope` and `Ident.highest_scope` is not so clear.
(Particularly the latter)